src/System.Numerics.Vectors/src/System/Numerics/VectorMath.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+#if HAS_INTRINSICS
```

Comment:
Instead of completely `#if` on this whole file, we should only include it when it is used.

So, we can remove this `#if`, and in the .csproj add it to `<Compile>` when `$(TargetsNetCoreApp)`.

Reply:
@eerhardt yeah I was thinking about it, but I guess I have to declare a new property $(IntrinsicsSupport) something like this in the csproj since we rely on HAS_INTRINSICS symbol